freelist.go

```diff
@@ -286,46 +237,6 @@ func (f *freelist) read(p *common.Page) {
 	}
 }
 
-// arrayReadIDs initializes the freelist from a given list of ids.
-func (f *freelist) arrayReadIDs(ids []common.Pgid) {
-	f.ids = ids
-	f.reindex()
-}
-
-func (f *freelist) arrayGetFreePageIDs() []common.Pgid {
-	return f.ids
-}
-
-// write writes the page ids onto a freelist page. All free and pending ids are
-// saved to disk since in the event of a program crash, all pending ids will
-// become free.
-func (f *freelist) write(p *common.Page) error {
```

Comment:
Please keep the `write` method in freelist.go because it has nothing to do with array or hashmap freelistType.

Reply:
copied too eagerly, moved it back where it was